src/sentry/templates/sentry/emails/error.html

```diff
@@ -159,7 +159,7 @@ <h3 class="title" style="margin-bottom: 10px">Suspect Commits</h3>
         <table class="table commit-table">
         {% for commit in commits %}
           <tr>
-            <td style="padding:0;width:32px;">{% email_avatar commits.author.name commits.author.email 32 %}</td>
+            <td style="padding:0;width:32px;">{% email_avatar commit.author.name commit.author.email 32 %}</td>
```

Comment:
Lol, is this the part that fixes the commit author avatar as seen in the visual snapshot diffs?

Reply:
Yes